src/app/src/util/util.js

```diff
@@ -243,3 +243,13 @@ export const getBBoxForArrayOfGeoJSONPoints = flow(
     featureCollection,
     bbox,
 );
+
+export const makeFacilityListItemsDetailLink = id => `/lists/${id}`;
+export const makePaginatedFacilityListItemsDetailLink = (id, page) => `/lists/${id}/${page}`;
+export const makePaginatedFacilityListItemsDetailLinkWithRowCount = (id, page, rowsPerPage) =>
+    `/lists/${id}/${page}/${rowsPerPage}`;
+
+export const makeSliceArgumentsForTablePagination = (page, rowsPerPage) => Object.freeze([
+    (page * rowsPerPage) + 1,
+    (page + 1) * rowsPerPage,
+]);
```

Comment:
Looking at this again I think this may be wrong, so I'll try to fix it. The `0` case begins the slicing at 1, which is wrong.